gordo/builder/mlflow_utils.py

```diff
@@ -141,7 +141,7 @@ def get_run_id(client: MlflowClient, experiment_name: str, model_key: str) -> st
         if experiment
         else client.create_experiment(experiment_name)
     )
-
+    # TODO Run check for runs taged with model key, or name runs by model key, to avoid re-logging same model
```

Comment:
You want to have this here still? Generally I guess, either fix, or make an issue. TODO statements are a bit of a bore.

Reply:
That was intentional, but now in hindsight I agree with you and disagree with my(past)self.